src/hooks/aelin/useAelinPoolStatus.ts

```diff
@@ -201,6 +202,9 @@ function useUserActions(
   derivedStatus: DerivedStatus,
 ): PoolAction[] {
   const { address: walletAddress } = useWeb3Connection()
+  const [balance] = useERC20Call(pool.chainId, pool.address, 'balanceOf', [
```

Comment:
Is there a way to avoid calling the `balanceOf `contract function here? This hook is often used and we've tried not to add requests to avoid a possible performance issue.